### ISSUE TITLE
Logging

### ISSUE THREAD

**Issue by midspace** [midspace/Space-Engineers-Admin-script-mod#102]
I've found we can write to the SE Log with the following:
` VRage.Utils.MyLog.Default.WriteLine("####### HELLO WORLD ######");`

**Comment by Spcemarine** [comment]
That might be very useful. Instead of writing the catched exceptions into a new file, we could use this as it is easier to find the file.

**Comment by zrisher** [comment]
I would argue that it's highly unusual for a mod to use the main SE log, and that proper separation of concerns dictates a separate log for each mod. But that's just me, I already have ready-made logger classes I work with so it's easy to take that position.

**Comment by Spcemarine** [comment]
I agree with you and we also have our logger classes so we don't really need it. But users sometimes have difficulties to find the files of our mod that's why I'd put it there. For mere debug information for us there is no need to write it to the SE log.

**Comment by midspace** [comment]
At the very least, we could write in the SE log that there is a another log for the admin helper at C:\blah\...

**Comment by midspace** [comment]
I can tell you what this is good for. Checking Threads.
As the Logging writes the Thread Id.

**Comment by Spcemarine** [comment]
And with WriteLineAndConsole(...) you can display some debug information on the ds console.
